packages/quantic/force-app/examples/main/lwc/exampleQuanticPlaceholder/exampleQuanticPlaceholder.js

```diff
@@ -0,0 +1,26 @@
+import {LightningElement, track} from 'lwc';
+
+export default class ExampleQuanticPlaceholder extends LightningElement {
+  @track config = {};
+  isConfigured = false;
+
+  pageTitle = 'Quantic Placeholder';
+  pageDescription = 'The Quantic Placeholder component is used internally to display a loading stencil for certain components.';
```

Comment:
ah yes well i liked the term stencil because that's what salesforce. calls them :P at first the component was named `quanticStencil` but I decided to change it because it might be confusing. I must've left this one in subconsciously because I still like the term haha. I'll change it to placeholder.